src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/scanContext.hpp

```diff
@@ -36,16 +36,18 @@ enum class ScannerType
     Unknown = 0,
     PackageInsert = 1,
     PackageDelete = 2,
-    Os = 3,
-    HotfixInsert = 4,
-    HotfixDelete = 5,
+    PackagesClear = 3,
+    Os = 4,
+    HotfixInsert = 5,
+    HotfixDelete = 6,
```

Comment:
I think we'll have a conflict here, with the PR #21118

Reply:
Hi @GabrielEValenzuela what if we have to do the same with hotfixes? I use PackagesClear to be more specific, so we can use something like HotfixesClear.

Reply:
Amazing idea! Could you do it on this PR?

Reply:
The scope is only for packages, we could add the hotfixes implementation later, or maybe I didn't understand the comment.

Reply:
Solved!